src/plugins/presentation_util/public/components/field_picker/field_picker.tsx

```diff
@@ -110,13 +115,14 @@ export const FieldPicker = ({
         placeholder: i18n.translate('presentationUtil.fieldSearch.searchPlaceHolder', {
           defaultMessage: 'Search field names',
         }),
+        disabled: Boolean(selectableProps?.isLoading),
       }}
       listProps={{
         isVirtualized: true,
         showIcons: false,
         bordered: true,
       }}
-      height={300}
+      height="full"
```

Comment:
I'm applying the height to the `EuiSelectable` via the `fieldPickerSelectable` class instead - this makes it so that switching from the loading state to showing the fields does not cause layout shifting:

**Old**

**New**

![Mar-30-2023 14-43-27](https://user-images.githubusercontent.com/8698078/228959733-4a857e5e-73c5-41c8-af9f-ec2c92e0efd0.gif)

Reply:
That looks so much better!